src/Core/src/Handlers/Picker/PickerHandler.Android.cs

```diff
@@ -165,5 +167,14 @@ static void Reload(IPickerHandler handler)
 		{
 			handler.PlatformView.UpdatePicker(handler.VirtualView);
 		}
+
+		void PlatformViewLayoutChange(object? sender, Android.Views.View.LayoutChangeEventArgs e)
+		{
+			if (PlatformView != null && VirtualView != null)
+			{
+				PlatformView.SetHeight(PlatformView.Height);
```

Comment:
Not sure I understand what this line is doing?

Reply:
Yes, this is a strange one, but it fixes it. For some reason, these heights have different values

Reply:
What are the different values? Is one really incorrect but still a valid size? Like before it is 30 but the correct height is 300, or is it 0 and 300?

Reply:
It is 0 and then 300